prog/dftb+/lib_geoopt/lbfgs.F90

```diff
@@ -139,7 +140,13 @@ module dftbp_lbfgs
     !> Is a line search used
     logical :: isLineSearch
 
+    !> Is the original line search used
+    logical :: isOldLSUsed
+
     !> line minimizer
+    type(TLineMin) :: lineMin
+
+    !> original internal line search
     type(TLineSearch) :: lineSearch
```

Comment:
It would be more appropriate to make them `allocatable` and allocate only the one which is needed.